2-mandatory/9-cheap-diner.js

```diff
@@ -19,7 +19,14 @@ chosenMeal(emptyArray)
 Should give the answer "Nothing :("
 **/
 
-function chooseMeal(mealArray) {}
+function chooseMeal(mealArray) {
+  if (mealArray.length === 0) return `Nothing :(`;
+  if (mealArray.length === 1) return mealArray[0].name;
+
+  return mealArray.sort((mealA, mealB) => {
+    return mealA.price - mealB.price;
```

Comment:
Same as above about the internal brackets!
`sort((a, b) => a.price - b.price)` works!